src/Raven.Server/Documents/Handlers/TimeSeriesHandler.cs

```diff
@@ -179,7 +179,7 @@ public async Task Read()
                     return;
                 }
 
-                var rangeResult = GetTimeSeriesRange(context, documentId, name, from, to, ref start, ref pageSize);
+                var rangeResult = pageSize > 0 ? GetTimeSeriesRange(context, documentId, name, from, to, ref start, ref pageSize) : new TimeSeriesRangeResult();
```

Comment:
Can GetTimeSeriesRange return empty result when pageSize is 0?

Reply:
yes